README.md

```diff
@@ -332,12 +332,21 @@ If you want to import uncompiled source and transpile it yourself (not recommend
 import { NFC } from 'nfc-pcsc/src';
 ```
 
-### Can I read a NDEF formatted tag?
+### How to read or write NDEF messages?
 
-**Yes, you can!** You can read raw byte card data with `reader.read` method, and then you can parse it with any NDEF parser, e.g. [TapTrack/NdefJS](https://github.com/TapTrack/NdefJS).
+**Yes, you can!** First install nfc-tools:
 
-**Psst!** There is also an example ([ndef.js](/examples/ndef.js)), but it is not finished yet. Feel free to contribute.
+```sh
+npm install nfc-tools --save
```

Comment:
The npm package name is (now?) `nfccard-tool` :)

Reply:
"now", yes.  
https://github.com/somq/nfccard-tool   
https://www.npmjs.com/package/nfccard-tool